jspoon/src/main/java/pl/droidsonroids/jspoon/HtmlField.java

```diff
@@ -38,7 +38,7 @@
 
     private void setLocaleFromTag(String localeTag) {
         if (localeTag.equals(Selector.NO_VALUE)) {
-            locale = Locale.getDefault();
+            locale = Locale.ROOT;
```

Comment:
jak nie przywrócisz to następne klasy odpalone w tym samym VM będą miały też zmienione, więc trzeba zapamiętać i przywrócić